src/User_Command.php

```diff
@@ -1030,7 +1030,7 @@ private static function validate_role( $role ) {
 	}
 
 	/**
-	 * Acommodate three different behaviors for wp_new_user_notification()
+	 * Acommodates three different behaviors for wp_new_user_notification()
```

Comment:
There was a typo in the original description.
This should be `Accommodates` (2 `c`s).

Reply:
Fair enough ;)

I've also updated the PR from latest `master`.